### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

**Comment by jamie-lemon** [comment]
Thanks for this, could you possibly redo this and target the 1.21 branch? ( Or documentation on master is a little out of date )

**Comment by kianmeng** [comment]
@jamie-lemon Done. Please check.

File: docs/recipes-general.rst [hunk 1]
```diff
@@ -203,7 +203,7 @@ If a clean, non-corrupt / decompressed PDF is needed, one could dynamically invo
  print pdf.Info
  # do further processing
 
-With the command line utility *pdftk* (`available <https://www.pdflabs.com/tools/pdftk-the-pdf-toolkit/>`_ for Windows only, but reported to also run under `Wine <https://www.winehq.org/>`_) a similar result can be achieved, see `here <http://www.overthere.co.uk/2013/07/22/improving-pypdf2-with-pdftk/>`_. However, you must invoke it as a separate process via *subprocess.Popen*, using stdin and stdout as communication vehicles.
+With the command line utility *pdftk* (`available <https://www.pdflabs.com/tools/pdftk-the-pdf-toolkit/>`_ for Windows only, but reported to also run under `Wine <https://www.winehq.org/>`_) a similar result can be achieved, see `here <http://www.over there.co.uk/2013/07/22/improving-pypdf2-with-pdftk/>`_. However, you must invoke it as a separate process via *subprocess.Popen*, using stdin and stdout as communication vehicles.
```
Comment (jamie-lemon) [inline docs/recipes-general.rst:206]: I think this diff is incorrect - `overthere` is part of a URL.

**Review by jamie-lemon — CHANGES_REQUESTED** [review]
Thank you. One small change required.

**Comment by julian-smith-artifex-com** [comment]
recheck

**Comment by julian-smith-artifex-com** [comment]
> Found via `codespell docs -L ist`
> 
> I have read the CLA Document and I hereby sign the CLA

It looks like the automated CLA system isn't recognising this as acceptance of the CLA.

I think it might be because of the other text in your message "Found via `codespell docs -L ist`".

Could you try again, this time posting just the acceptance text in a message here?

**Comment by kianmeng** [comment]
@julian-smith-artifex-com I've updated the PR comment to only CLA agreement line. Please check again.

**Comment by julian-smith-artifex-com** [comment]
> @julian-smith-artifex-com I've updated the PR comment to only CLA agreement line. Please check again.

Unfortunately that doesn't seem to have worked. Could you try  creating a new message with the CLA text?

**Comment by kianmeng** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by julian-smith-artifex-com** [comment]
recheck

**Review by jamie-lemon — APPROVED** [review]

**Comment by jamie-lemon** [comment]
recheck

**Pull request merged**